test/integration/targets/wait_for_connection/aliases

```diff
@@ -0,0 +1,2 @@
+posix/ci/group2
+windows/ci/group2
```

Comment:
I don't really see why that is important. That is the default in VM.
Ironically PEP8 is requiring me to remove the trailing newline, and you are asking me to add one :-)